client/src/pages/Settings.js

```diff
@@ -90,6 +94,14 @@ const Settings = () => {
     }
   };
 
+  window.addEventListener(
+    "beforeunload",
+    event => {
+      axios.post(`/queue/${localStorage.getItem("email")}`);
+    },
+    false
+  );
+
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener 

Maybe try using the options object, setting "once" to true?

Reply:
changed it to true, and it seems to be being fired once now. THANK YOU!